doc/classes/ProjectSettings.xml

```diff
@@ -2930,7 +2930,7 @@
 		</member>
 		<member name="rendering/viewport/hdr_2d" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], enables [member Viewport.use_hdr_2d] on the root viewport. 2D rendering will use an high dynamic range (HDR) format framebuffer matching the bit depth of the 3D framebuffer. When using the Forward+ renderer this will be an [code]RGBA16[/code] framebuffer, while when using the Mobile renderer it will be an [code]RGB10_A2[/code] framebuffer. Additionally, 2D rendering will take place in linear color space and will be converted to sRGB space immediately before blitting to the screen. Practically speaking, this means that the end result of the Viewport will not be clamped into the [code]0-1[/code] range and can be used in 3D rendering without color space adjustments. This allows 2D rendering to take advantage of effects requiring high dynamic range (e.g. 2D glow) as well as substantially improves the appearance of effects requiring highly detailed gradients.
-			[b]Note:[/b] This setting will have no effect when using the GL Compatibility renderer as the GL Compatibility renderer always renders in low dynamic range for performance reasons.
+			[b]Note:[/b] This setting will have no effect when using the Compatibility renderer as the Compatibility renderer always renders in low dynamic range for performance reasons.
```

Comment:
```suggestion
			[b]Note:[/b] This setting will have no effect when using the Compatibility renderer as it always renders in low dynamic range for performance reasons.
```

Reply:
One could argue "it" is ambiguous and could refer to "This setting", but it obviously doesn't because otherwise this setting would be useless.

Reply:
Could alternatively use "Compatibility rendering **which** always renders".